vdbench_wrapper/Dockerfile

```diff
@@ -0,0 +1,14 @@
+FROM quay.io/avili/vdbench:1.0.1
+
+USER root
+RUN dnf install --nodocs -y python3-numpy
+RUN dnf clean all
```

Comment:
not clear why this is needed either.   All that does is delete caches right?  We want to exploit the cache for reasons you'll soon discover ;-)

Reply:
I delete it.